templates/cassiopeia/scss/vendor/choicesjs/choices.scss

```diff
@@ -85,6 +85,21 @@
     }
   }
 }
+.choices__list--multiple .choices__item {
+  border-radius: 5px;
+  font-size: inherit;
+  font-weight: 500;
+  background-color: var(--info);
+  border: 1px solid $white;;
+  color: $white;
+
+  &.is-highlighted {
+    color: white;
```

Comment:
Won't this lead to the same SCSS linter errror as I had fixed with #238 ? I.e. shouldn't this be `$white`and not `white`?

Reply:
fixed